theme/static/css/custom.css

```diff
@@ -385,3 +385,15 @@ section#pricing{
     z-index: -999999;
     opacity: 0;
 }
+
+.dropdown-submenu > a::after { margin-top: 15px; }
+.navbar-event .dropdown .dropdown-menu li .dropdown-menu  {
+    left: 100%;
+    top: 4px;
+}
+
+.navbar-event .dropdown .dropdown-menu > li > a:hover, .navbar-event .dropdown .dropdown-menu > li > a .active, .navbar-event .dropdown .dropdown-menu > li:hover > a { color: #4CAF50; }
+@media (max-width: 767px){
```

Comment:
@waqasahmad12345 You have added another media query block with 767. There is already a block for 767px, please add your chnages to that block